index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+<header id = "header">
+    <img src = "/images/company-logo.png" alt = "Company Logo" id = "header-img">
+    <nav id = "nav-bar">
+          <button class = "nav-link" href = "#Features">Features</button>
+          <button class = "nav-link" href = "#How_It_Works">How It Works</button>
+          <button class = "nav-link" href = "#Pricing">Pricing</button>
+    </nav>
+   
+   
+    
+</header>
+<div class="div-class">
+  <h1>Prototype Title</h1>
+<video id = "video" controls width = "250">
+<source src = "images/video.mp4" type="video/mp4">
+</video>
+<form id = "form" method="POST" action="https://www.freecodecamp.com/email-submit">
+    <br>
+    <br>
+    <div id = "div-input">
+<input type = "email" id = "email" placeholder="Insert e-mail here..." name = "email">
+<input id="submit" type="submit" value="submit">
+</div>
```

Comment:
Although I like the way you contained this, you should use semantic HTML tags to contain this section. Please use <section> tag to make your code better to read and function.

Reply:
- [ ] You started you comment in a polite manner it is very good 👍🏻 , however, I think that is not very clear, because you requested to add a tag, but what tag or tags is not clarified :grin:

- This is a clear comment: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?

- This is a vague comment: The README is not descriptive, why don't you make it better?

Reply:
Hi, it seems that using the < and > make the word disappear! I'm glad we found this early because I did mention to use section but since I used the open and closing symbols it vanished.